language-configuration.json

```diff
@@ -16,7 +16,7 @@
     ["[", "]"],
     ["(", ")"],
     ["|", "|"],
-    ["\"", "\""]
+    ["\"", "\""],
```

Comment:
Huh, I'm surprised this didn't cause problems before, was there an error reported somewhere that I missed?